tools/autograph-monitor/contentsignature.go

```diff
@@ -77,7 +77,11 @@ func verifyContentSignature(response signatureresponse) error {
 
 func getX5U(x5u string) (certs []*x509.Certificate, err error) {
 	log.Printf("Retrieving X5U %q", x5u)
-	resp, err := http.Get(x5u)
+	c := &http.Client{}
+	t := &http.Transport{}
+	t.RegisterProtocol("file", http.NewFileTransport(http.Dir("/")))
```

Comment:
can we disable this for prod?

Reply:
Does it matter if we do? The X5U is controlled by autograph and this is a monitoring function.

Reply:
oh right, so it'll only have access to the lambda context, which can only sign the hardcoded string. fine by me then.